angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -181,6 +181,7 @@
                 <nimble-table-column-text field-name="stringValue2">String 2</nimble-table-column-text>
             </nimble-table>
             <nimble-button class="add-table-row-button" (click)="onAddTableRow()">Add row</nimble-button>
+            <p><nimble-anchor href="#/tabledemo">Table control demo page (full height)</nimble-anchor></p>
```

Comment:
We shouldn't define multiple routes because we can't replicate it in Blazor because Blazor does not do hash-based routing (which we need on GitHub Pages).

Could maybe put it in a drawer or something if a tall demo is useful

Reply:
Yea from that proposal we would be rebranding these as the kitchen sink examples that are primarily a developer tool to integrate with storybook publishing / just a quick way for clients to have an overview of all the controls. I don't think we would do the same trick to ship all our `/examples` in every storybook publish to chromatic like we are doing now. But we would ship them to the published GitHub Pages.

This perf examples would move to `/examples` and there I would be less concerned about parity of examples across frameworks.

> This is a frustrating limitation, especially since this new example isn't really meant to be Angular-specific, it's more of a performance / interaction example.

Yea I don't think we should have the kitchen sink examples become kinda large monolithic dumping grounds of experiments. I'm thinking those can be smaller in narrowly focused `/examples`. 

I don't feel quite ready to make that move yet because I'm not sure of the right pattern to get Angular to build that efficiently. My concern is that the base overhead of angular builds seems really high so lots of small examples are going to take a long time to build but I think is ultimately the right move for, ie making stackblitzs from paths with smaller self contained easier to digest examples.

I added a note to the examples HLD for including a table performance example (if we think that will address the concern for removing it from the kitchen sink example for now): https://github.com/ni/nimble/pull/827/files#r1086053817

Reply:
Took out this separate table demo page for now